coops-ui/package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "vue-cli-service lint",
     "lint:nofix": "vue-cli-service lint --no-fix",
     "test:e2e": "vue-cli-service test:e2e",
-    "test:unit": "vue-cli-service test:unit",
+    "test:unit": "vue-cli-service test:unit --testPathPattern=*",
```

Comment:
what's the purpose of the new flag?

Reply:
It allows a developer to run the tests in a single file.
I kept having to look up the parameter name so I just left it in package.json with the '*' pattern... It's now easy to replace that with, for example, "AnnualReport", to run those tests only.